imageroot/actions/import-module/40Migration

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+set -e -o pipefail
+exec 1>&2 # Redirect any output to the journal (stderr)
+
+# Prepare an initialization script that restores the dump file
+mkdir -vp initdb.d
+mv -v roundcubemail.sql initdb.d
+#do the bash file to restore and exit once done
+cat - >initdb.d/roundcubemail_restore.sh <<'EOS'
+# Read dump file from standard input:
+cd docker-entrypoint-initdb.d
+mysql -uroundcubemail -proundcubemail roundcubemail < roundcubemail.sql
+# Print additional information:
+mysql --version
+# The script is sourced, override entrypoint args and exit:
+set -- true
+exit 1
+EOS
+# file must be executable
+chmod a+x initdb.d/roundcubemail_restore.sh
```

Comment:
The image entrypoint script includes (`source` aka `.`) .sh files if they are not flagged _executable_.

The `set -- true` line should cause the entrypoint exit by itself, after finishing the load of init scripts. By contrary `exit 1` forces the script termination with error, preventing any further operation. This is an unexpected behavior IMO... Why is it arranged like that?

```suggestion
set -- true
EOS
```

Reply:
the interactive container is not stopped at the end, it is not nice but the exit 1 stops the container